### PR TITLE
Fix gce bootstrap script guide

### PR DESCRIPTION
**Pull request by theidexisted**
The git cloned repo lacks the version file, which leads to a script fail

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature
> /kind flake

/kind bug

**What this PR does / why we need it**:
Fix gce bootstrap script guide 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```

```

### CODE REVIEW

**Comment by theidexisted** [comment]
EasyCLA

File: docs/getting-started.md [hunk 1]
```diff
@@ -44,10 +44,11 @@ repository](https://github.com/kubernetes/kubernetes) will automatically format
 and mount the requested Local SSDs, so you can deploy the provisioner with the
 pre-generated deployment spec and skip to [step
 4](#step-4-create-local-persistent-volume-claim), unless you want to customize
-the provisioner spec or storage classes.
+the provisioner spec or storage classes. We need to download the release from https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG, the latest release at the time of writing is v1.23.5.
```
Comment (cofyc) [inline docs/getting-started.md:47]: we can ask people to download release tar file, but I suggest don't write specific version in docs.

Comment (theidexisted) [reply docs/getting-started.md:47]: It's intended to show as a demonstration with the latest build. Could you please kindly provide the suggested change from your side?

Comment (cofyc) [inline docs/getting-started.md:47]:
```suggestion
the provisioner spec or storage classes. We need to download the release from https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.
```

**Comment by cofyc** [comment]
LGTM
@theidexisted can you squash to one commit?

**Comment by theidexisted** [comment]
> LGTM @theidexisted can you squash to one commit?

Thanks! Github supports merge with squash, could you please use this type of merge? (I just edited the file on the web)

**Pull request merged**